test/cluster_test.go

```diff
@@ -894,6 +894,30 @@ func negativeDeployClusterValidationTest(
 					)
 				},
 			)
+
+			Context(
+				"InvalidDNSConfiguration", func() {
+					It(
+						"InvalidDnsPolicy: should fail when dnsPolicy is set to 'Default'",
+						func() {
+							aeroCluster := createDummyAerospikeCluster(clusterNamespacedName, 1)
+							aeroCluster.Spec.PodSpec.DNSPolicy = v1.DNSDefault
+							err := deployCluster(k8sClient, ctx, aeroCluster)
+							Expect(err).Should(HaveOccurred())
+						},
+					)
+
+					It(
+						"MissingDnsConfig: should fail when dnsPolicy is set to 'None' and no dnsConfig given",
```

Comment:
Add a positive test case where dnsPolicy is `none` and dnsConfig is given

Reply:
Done